File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<int> dp(26, 0);
        for(char c: s){
            dp[c - 'a']++;
        }
        for(char c: t){
            dp[c - 'a']--;
        }
        for(int d: dp){
            if(d !=0) return false;
        }
        return true;
    }
};

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.py
import heapq as hp
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        heap = []
        for num in nums:
            if len(heap) < k:
                hp.heappush(heap, num)
            else:
                hp.heappushpop(heap, num)
        return heap[0]